faq/general.md

```diff
@@ -1,6 +1,6 @@
 # The Big Picture
 
-## Should dapp developers run their own relay server? <a id="do_i_have_to_run_a_relayer?"></a>
+## Should dapp developers run their own relay server? <a id="do_i_have_to_run_a_relayer"></a>
```

Comment:
true, but need to "relax" the "highly  recommended". dApps can work without a relayer at all.